frontend/src/components/ManagePermissionsTable/index.vue

```diff
@@ -448,6 +449,33 @@ const highlightNewUserAccessRow = (
             return undefined;
     }
 };
+
+/**
+ * DataTable 'Organization' column display expression.
+ * @param {FamApplicationUserRoleAssignmentGetSchema | FamAccessControlPrivilegeGetResponse} data - provided
+ *        datatable data to extract and format forest client information for the 'Organization' column. Only
+ *        user table and delegated admin table have this column.
+ */
+const displayForestClient = (
+    data:
+        | FamApplicationUserRoleAssignmentGetSchema
+        | FamAccessControlPrivilegeGetResponse
+) => {
+    const forestClientData = data.role.forest_client;
+    if (props.tableType === ManagePermissionsTableEnum.AppUser) {
+        // Display formatted forest client display name.
+        return forestClientData
+            ? formatForestClientDisplayName(
+                  forestClientData.forest_client_number,
+                  forestClientData.client_name
+              )
+            : "";
+    } else {
+        // For delegated admin data.
+        // TODO: No client name available from backend yet, implement soon. Only display client number. # noqa NOSONAR
+        return forestClientData?.forest_client_number;
+    }
+};
```

Comment:
Could you move this to the utils.ts under the same folder? so the file doesn't get too long

and perhaps rename it to getOrganizationName?

Reply:
Thank for the suggestion Craig! This is moved.